x-pack/plugin/runtime-fields/src/test/java/org/elasticsearch/xpack/runtimefields/mapper/ScriptFieldTestCase.java

```diff
@@ -15,7 +15,7 @@
 import java.io.IOException;
 import java.util.Map;
 
-public abstract class ScriptFieldScriptTestCase<T> extends ESTestCase {
+public abstract class ScriptFieldTestCase<T> extends ESTestCase {
```

Comment:
should this rather be FieldScriptTestCase?

Reply:
That sounds better! Please rename it too!